### DISCUSSION TITLE
求解器失效

### DISCUSSION THREAD

**Discussion by 1fan-lee** [OpenRadioss/OpenRadioss#3164] [Q&A]
昨天使用脚本运算RAD文件时能正常运算，还生成了完整的结果文件，但是今天再使用时显示失效，也没有error，请问是什么原因呢
![e44877245c989463b6e74a8d17462823](https://github.com/user-attachments/assets/3ea3e241-effe-4d04-9f83-47da266e619a)

**Comment by MarianBulla** [comment]
Hello @1fan-lee, 
thanks for you interest in OpenRadioss.

Can you place your question in english, please ?

So, I (and/or the community) will be able to help you :-) 

Thanks and best regards,
Marian

**Reply by 1fan-lee** [reply]
Yesterday, when I used the script to calculate the RAD file, it was able to calculate normally, and the complete result file was generated, but when I used it again today, it showed invalidity, and there was no error, what is the reason?
![389747841-3ea3e241-effe-4d04-9f83-47da266e619a](https://github.com/user-attachments/assets/d2f2c6b5-d8ae-467f-a06c-c185dd53c654)

**Comment by OlivierWienholtz** [comment]
Hello,
Unfortunately, I cannot read the error message.
One possible reason : Intel MPI on Windows or OpenMPI on Linux must be available before launching the script.

Let me know if this fixes.
